core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -326,6 +326,38 @@ static function (array $value): array {
                     ->prototype('scalar')->end()
                     ->defaultValue(['jpg', 'jpeg', 'gif', 'png', 'tif', 'tiff', 'bmp', 'svg', 'svgz', 'webp'])
                 ->end()
+                ->arrayNode('preview')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('target_dir')
+                            ->info('The target directory for the cached previews.')
+                            ->example('%kernel.project_dir%/assets/previews')
+                            ->cannotBeEmpty()
+                            ->defaultValue(Path::join($this->projectDir, 'assets/previews'))
+                            ->validate()
+                                ->always(static fn (string $value): string => Path::canonicalize($value))
+                            ->end()
+                        ->end()
+                        ->integerNode('default_size')
+                            ->min(1)
+                            ->max(65535)
+                            ->defaultValue(512)
+                        ->end()
+                        ->integerNode('max_size')
+                            ->min(1)
+                            ->max(65535)
+                            ->defaultValue(1024)
+                        ->end()
+                        ->booleanNode('fallbacks')
```

Comment:
Should we name this `fallback_images` or `show_fallback_image`?

Reply:
Without checking into the code, shouldn't this be a developer decision for the element that creates the preview? I mean a download element might _know_ it always wants fallbacks, but other elements might not? Or something the user decides through a checkbox. It doesn't sound right to enable or disable this globally in the application.

Reply:
We could do both, an option for the FallbackProvider as well as the current config. But I think disabling these images for the whole project is still a valid use case.

Reply:
How about naming it `enable_fallback_images` or `enable_fallbacks`?

Reply:
Could be as "simple" as having a dropdown rather than a checkbox for preview. `disable`, `enable_with_fallback` and `enable_without_fallback`?

Reply:
I like `enable_fallback_images`. I also like @Toflar's suggestion if you decide to go this way.

Reply:
The global fallback is required anyway. E.g. to decide for the file manager.

Reply:
> E.g. to decide for the file manager.

Which could/should be a user config, similar to "show preview images"? 🤷

Reply:
Renamed to `enable_fallback_images` in af2ae0f9022b5b555b648e547a5b9d46cb00774f